addons/website/static/src/js/menu/seo.js

```diff
@@ -660,6 +665,7 @@ var SeoConfigurator = Dialog.extend({
                 canEditTitle: self.canEditTitle,
                 canEditDescription: self.canEditDescription,
                 isIndexed: self.isIndexed,
+                previewDescription: _t('The description will be generated by search engines based on page content unless you specify one.'),
```

Comment:
Why is this C/C multiple times in the code now and much of the code changed while the specs seems to only tell to change the sentence ?

Reply:
I adapted the code to be able to have two different descriptions as requested. Concerning the sentence you mention here, this is not perfectly the same one but I optimized a bit what I did before.